bin/commands/runs.js

```diff
@@ -61,6 +61,9 @@ module.exports = function run(args, rawArgs) {
     markBlockStart('setConfig');
     logger.debug('Started setting the configs');
 
+    // set cypress config filename
+    utils.setCypressConfigFilename(bsConfig, args);
```

Comment:
Let's move our code in end in this scenario?

Reply:
That would require moving more code around. This change shouldn't affect handling of other products.